sdks/python/apache_beam/coders/observable.py

```diff
@@ -20,6 +20,11 @@
 
 For internal use only; no backwards-compatibility guarantees.
 """
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
<!--new_thread; commit:cffe1c29cd6256ad280677ba9c9aeb3eeffe7269; resolved:0-->
Which of these is really necessary?  Do we use division and print in this file?